spec/models/form_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+describe Form, type: :model do
+  let(:form) { create(:form) }
+
+  it 'belongs to an event' do
+    expect(form.event).to be_an_instance_of(Event)
+  end
+
+  context 'with scopes' do
+    it 'defaults to active forms' do
+      active = create(:form, status_id: 1)
+      create(:form, status_id: 0)
+      expected_id = active.id
+      expect(described_class.all.pluck(:id)).to eq([expected_id])
+    end
+
+    it 'does not include results outside of date range' do
```

Comment:
I'd like to see a couple more tests here to prove that query works how we expect.

Does it work when the there's an actual date range for start and end? (The current factory defaults just set both the start and end to the same value)

Does it filter out forms that are not yet effective?

I don't think that your code is wrong, but there's a lot of important functionality hidden in those few lines of query so I want make sure that we get it right and don't accidentally break it in the future.

Reply:
I will add some more tests with bigger date ranges and also add a test where the form is within the date range (that is also being tested in agencies_controller_spec.rb)

> Does it filter out forms that are not yet effective?

Can you provide an example of what you want tested here?  It currently tests the status_id of 0 and 1.

Reply:
You covered it. Tests look good now. :+1: